denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/functions/mod_commands.dsc

```diff
@@ -14,12 +14,12 @@ mod_server_kick_task:
     - define moderator <tern[<context.source_type.is[==].to[PLAYER]>].pass[<player.uuid>].fail[Server]>
     - define player <server.match_offline_player[<context.args.first>]>
```

Comment:
This isn't making any effort to verify the player or that the first argument is valid
```suggestion
    - if !<context.args.size.is_empty>:
      - define user <context.args.first>
      - inject player_verification_offline
      - define player <[user]>
    - else:
      - define reason "You must specify a player."
      - inject command_error
```